db/re/item_db_equip.yml

```
@@ -177855,6 +183777,30 @@ Body:
              bonus2 bResEff,Eff_Stun,10000;
          }
       }
+  - Id: 460057
+    AegisName: aegis_460057
+    Name: Spenta Armaiti
+    Type: Armor
+    Weight: 300
+    Defense: 60
+    Slots: 1
+    Locations:
+      Left_Hand: true
+    ArmorLevel: 1
+    EquipLevelMin: 100
+    Refineable: true
+    View: 4
+    Script: |
+      .@r = getrefine();
+      bonus bUnbreakableShield;
+      bonus2 bSubClass,Class_Boss,40;
+      bonus2 bSubClass,Class_Normal,40;
+      bonus bMaxHPrate,3*.@r;
+      bonus bMaxSPrate,3*.@r;
+      if (.@r>=5)
+         bonus2 bSubEle,Ele_Holy,25-(getskilllv("CR_TRUST")*25);
```

Are you sure about `getskilllv("CR_TRUST")*25` ?
Maybe I read it wrong, for me it should be 
```suggestion
         bonus2 bSubEle,Ele_Holy,25-(getskilllv("CR_TRUST") > 0 ? 25 : 0);
```

---

Thanks!